src/pages/_middleware.tsx

```diff
@@ -0,0 +1,43 @@
+import { NextResponse } from "next/server";
+
+export function middleware() {
+  const ContentSecurityPolicy = `
+  frame-src 'self';
+  frame-ancestors 'none';
+  object-src 'self';
+  base-uri 'self';
+  script-src 'self' 'unsafe-eval' 'unsafe-inline' https://plausible.io/js/plausible.js;
```

Comment:
So one of the really cool things `next-plausible` does is include a forwarder to fetch this JS through the local domain. Main benefit of that is to prevent adblockers from blocking by URL. I'm guessing that sourcing this here will allow adblocks to eat all our metrics 😅

Reply:
This section wouldn't be sourcing plausible, rather it's saying that the only script allowed to be launched on this page is the one sourced from plausible. https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src

This will also improve SEO scores, as Google's crawlers can see the site is safe and explicitly doesn't allow any possibly unknown or harmful scripts.